pycontw/pycon.tw
# feat(core/EventInfo): add willing_to_attend_in_person field

## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies


## Description
**Describe what the change is**
You don't need to explain HOW you change the code. Your code, including comments in the code should explicitly
self-describe enough about how you change the code.

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Go to `proposals/tutorial/submit/`
2. Check the displayed page.

## Expected behavior
It would appear the field "attending PyCon TW in person" on the creating proposal/tutorial page.

## More Information

![image](https://user-images.githubusercontent.com/18432820/219711564-ba8aa9e8-d983-4817-9f8c-bd00ce780189.png)
